src/test/resources/pages/InstructorCourseStudentDetailsRegistered.html

```diff
@@ -196,7 +196,7 @@
 	<script src="/js/contextualcomments.js" type="text/javascript"></script>
 	<script type="text/javascript">
 	    var isShowCommentBox = false;
-	    var commentRecipient = "";
+        var commentRecipient = '';
```

Comment:
Oh wait, this is HTML. I guess this is regenerated by God mode this way? Then let it be. My bad.